docs/client/nakama/relay.mdx

```diff
@@ -148,3 +148,59 @@ catch (ApiResponseException ex)
     Debug.LogFormat("Error: {0}", ex.Message);
 }
 ```
+
+### Viewing Receipts and Events
+
+Game clients can view [events](/cardinal/game/system/event) emitted in systems as well as the results of iterating over [messages](/cardinal/game/message#iterating-over-messages) by listening for notifications from Nakama.
+
+#### Example
+
+The code to receive notifications will be different depending on the game engine/client you are using. Learn more about how to receive notifications for your game client at [Nakama docs](https://heroiclabs.com/docs/nakama/concepts/notifications/#receive-notifications).
+
+Here are structs that can be used to deserialize notification information. Note, the Receipt.Result field is an arbitrary object that will depend on the exact messages you've defined in your game.
+
+```csharp Unity/C# Example
+using System;
+using System.Text.Json.Serialization;
+
+public class Event
+{
+    [JsonPropertyName("message")]
+    public string Message { get; set; }
```

Comment:
JsonPropertyName would be really nice to have, but unfortunately we can't use it.  This will work though:

[Serializable]
[SuppressMessage("ReSharper", "InconsistentNaming")]
public struct TxHash
{
    public string txHash;
}